src/mixins/filing-mixin.ts

```diff
@@ -88,4 +91,54 @@ export default class FilingMixin extends Vue {
   hasFilingCode (filingCode: FilingCodes): boolean {
     return !!this.filingData.find(o => o.filingTypeCode === filingCode)
   }
+
+  /**
+   * Builds an Incorporation Application Correction filing body from IA filing. Used when creating a IA Correction.
+   * @returns the IA Correction filing body
+   */
+  buildIaCorrectionFiling (iaFiling: IncorporationFilingIF): CorrectionFilingIF {
+    const correctionFiling: CorrectionFilingIF = {
+      header: {
+        name: FilingTypes.CORRECTION,
+        certifiedBy: iaFiling.header.certifiedBy,
+        date: this.getCurrentDate
+      },
+      business: {
+        legalType: iaFiling.business.legalType,
+        identifier: iaFiling.business.identifier,
+        legalName: iaFiling.business.legalName
+      },
+      correction: {
+        correctedFilingId: iaFiling.header.filingId,
+        correctedFilingType: FilingTypes.INCORPORATION_APPLICATION,
+        correctedFilingDate: iaFiling.header.date,
+        comment: '',
+        incorporationApplication: {
```

Comment:
I'm thinking that we don't need all the interfaces you've ported over, since you can use assign the filing object here.
Explanation: other than here, we don't need to know the structure of the IA. I think we can just handle it as an object of type "any" for the purposes of creating a draft correction filing.

Reply:
For sure, it isn't 100% required but i don't think having the interfaces here is a bad thing. Going forward into Alterations or Company profile view etc we may find that we are wanting these interfaces. 
In a perfect world, we are importing these interfaces, but since we are not there yet, is it ok to leave these in?

Reply:
I think it's better to wait until we can import them. It will lessen the risk of have multiple copies that possibly diverge (thus creating more work to fix later). Plus we're not _really_ using them atm.

Reply:
Ok, fair points! No worries at all, will update this 👍

Reply:
Updated in next commit

Reply:
Could you simply do this here?
```
   incorporationApplication: iaFiling.incorporationApplication,
```

Reply:
I'm not sure, i will try!

Reply:
Yay :)